test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.cs

```diff
@@ -482,6 +483,50 @@ public void AreEqualShouldFailWhenNotEqualTypeWithMessage()
         StringAssert.Contains(ex.Message, "A Message");
     }
 
+    [TestMethod]
+    public void AreEqualShouldIgnorCaseBasedOnTheCurrentCulture()
```

Comment:
It would be nice to test that with English culture it's equal (you can either duplicate the test or take some `DataRow` with culture as string and expected result as bool). If you go with only one test, then test method name should be updated.